docs/topics/media-pipeline.rst

```diff
@@ -332,6 +346,23 @@ To handle media redirections, set this setting to ``True``::
 
     MEDIA_ALLOW_REDIRECTS = True
 
+.. _topics-media-pipeline-cache:
+
+Limiting cache size
+-------------------
+
+.. setting:: MEDIA_CACHE_SIZE
+
+By default media pipeline caches meta information
+(url, path and checksum) for all downloaded files and images.
+This might consume noticeable amount of memory for large-scale crawls
+with millions of items downloaded. Set it to the number of most recently
+used items you wish to keep::
+
+    MEDIA_CACHE_SIZE = 10000
+
+Zero is also an allowed value, this will disable the cache completely.
```

Comment:
Right, thanks for the pointers! Actually, in this case zero is not the default, default is an empty value that would be treated differently from zero. Zero is no cache, empty is unlimited cache. Does this sound confusing?